content/en/blog/_posts/XXXX-XX-XX-fine-grained-supplementalgroups-control-beta/index.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: blog
+title: 'Kubernetes 1.33: Fine-grained SupplementalGroups control graduates to beta'
+date: XXXX-XX-XX
+slug: fine-grained-supplementalgroups-control-beta
+author: >
+  Shingo Omura (LY Corporation)
+
+---
+
+Kubernetes 1.31 introduced new field `SupplementalGroupsPolicy` in Pod's Security Context to improve supplemental groups handling in containers in Pods.
+
+In Kubernetes 1.33, the feature graduates to beta and the corresponding feature gate (`SupplementalGroupsPolicy`) gets enabled by default.
+
+Please be aware that this beta release contains some behavioral breaking change. See [The Behavioral Changes Introduced In Beta](#the-behavioral-changes-introduced-in-beta) and [Upgrade Considerations](#upgrade-consideration) sections for details.
+
+## Motivation: Implicit group memberships defined in `/etc/group` in the container image
+
+Although this behavior may not be popular with many Kubernetes cluster users/admins, kubernetes, by default, _merges_ group information from the Pod with information defined in `/etc/group` in the container image.
```

Comment:
I don't know if `popular` is the correct wording here. maybe `Although the majority of Kubernetes cluster admins/users may not be aware, kubernetes, by default...`

Reply:
fixed in [caff013](https://github.com/kubernetes/website/pull/49944/commits/caff01322db36f60553e11d26aff2e78916f8ab9)